galleries/examples/images_contours_and_fields/image_exact_placement.py

```diff
@@ -0,0 +1,167 @@
+"""
+=========================================
+Placing images, preserving relative sizes
+=========================================
+
+By default Matplotlib resamples images created with `~.Axes.imshow` to
+fit inside the parent `~.axes.Axes`.  This can mean that images that have very
+different original sizes can end up appearing similar in size.
+
+Sometimes, however,  it is desirable to keep the images the same relative size, or
+even to make the images keep exactly the same pixels as the original data.
+Matplotlib does not automatically make either of these things happen,
+but it is possible with some manual manipulation.
```

Comment:
Optional: Slightly more concise wording:
```suggestion
This example shows how to keep the images the same relative size, or
how to make the images keep exactly the same pixels as the original data.
```